packages/_shared/src/FieldConnector.ts

```diff
@@ -61,7 +62,7 @@ export class FieldConnector<ValueType> extends React.Component<
       externalReset: 0,
       value: initialValue,
       lastRemoteValue: initialValue,
-      disabled: props.isInitiallyDisabled,
+      disabled: props.isDisabled ?? props.isInitiallyDisabled,
```

Comment:
@z0al How does this look?

Reply:
This will have the same effect as `isInitiallyDisabled`. If you want to keep the field disabled no matter what if `isDisabled` is passed then you need to conditionally [subscribe to `isDisabledChanged` updates](https://github.com/contentful/field-editors/blob/32bfe1dfaf6d976c69690a99b69b5beff570eba6/packages/_shared/src/FieldConnector.ts#L106-L110) here.

```javascript
if(isDisabled) { 
    this.unsubscribeDisabled = field.onIsDisabledChanged((disabled: boolean) => {
      this.setState({
        disabled,
      });
    });
}
```

However, I'd encourage you to look into why `isInitiallyDisabled` didn't work for your use case first

Reply:
Hey @z0al we took a slightly different approach, but I think would work a bit better because it would still allow for the listener to be subscribed while also allowing for a manual override based on the `isDisabled` prop.

The problem is that the `field.onIsDisabledChanged` method is being provided by the `app sdk`, which takes the state of of the field from the web app. The web app considers the field to be enabled, because it just sees it as rendering an app. It doesn't understand what is happening inside the app. So when the `field.onIsDisabledChanged` gets called on the initial render it immediately switches the field editor within the app to enabled.